azerothcore/azerothcore-wotlk
# fix(#2147): enrage and some false-negative spells

## CHANGES PROPOSED:

Add TARGET_UNIT_CONE_ENEMY_54 to the _isPositiveTarget() method

## ISSUES ADDRESSED:

Trying to fixing #2147 in a proper way, I've made this research: https://github.com/azerothcore/azerothcore-wotlk/issues/2147#issuecomment-647829243

It looks like that the "TARGET_UNIT_CONE_ENEMY_54" value is not considered within the _isPositiveTarget check. But it should.

This PR can also fix other spells that could be related:

![image](https://user-images.githubusercontent.com/147092/85348106-d49b6980-b4fa-11ea-9729-a095c92dfe0e.png)


## TESTS PERFORMED:

No tests performed


## HOW TO TEST THE CHANGES:

the Magdamar enrage should be still dispellable.

Go to the molten core and wait for Magdamar enrage. Then use Tranquillizing Shot or other similar spells to remove the magdamar buff.

## KNOWN ISSUES AND TODO LIST:

https://github.com/azerothcore/azerothcore-wotlk/issues/2129

https://github.com/azerothcore/azerothcore-wotlk/issues/2147


## Target branch(es):
